### ISSUE TITLE
TextField A11y Pass

### ISSUE THREAD

**Issue by TestAccessibilityGIT** [OfficeDev/office-ui-fabric-react#7338]
Tracking all of the issues brought up with the 12/6 accessibility pass on TextField control 
Details on individual bugs can be found on [SharePoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7B791f6669-085c-48de-a524-cf9ead8edf0d%7D&action=default&uid=%7B791F6669-085C-48DE-A524-CF9EAD8EDF0D%7D&ListItemId=1599&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble)
  
 - [ ] Bug1: [Accessibility][Keyboard Navigation]: Landmarks must have a unique role or role/label combination(aria-label OR aria-labelledby to make landmarks distinguishable) 
- [ ] Bug2: [Accessibility][Programmatic access]: Labels are not associated with form elements. 
- [ ] Bug3: [Accessibility][Programmatic access]: Narrator narrates 'Underscore' multiple times on 'With number mask'  and 'With input mask' text field. 
- [ ] Bug4: [Accessibility][Programmatic access]: length of inputs/digits to be entered is not defined in code for 'With number mask' text field. 
- [ ] Bug5: [Suggestion][Accessibility][Usable]: Narrator speaks 'Error Message' prior to the control information. " 
- [ ] Bug6: [Accessibility][Programmatic access]: Narrator does not read the info. message in scan mode.  
- [ ] Bug7: [Accessibility][Keyboard navigation]: Keyboard focus does not jump to 'Number Text Field with valid initial value' text field after selecting 'restore' button. 
- [ ] Bug8: [Suggestion][Accessibility][Usable]: Narrator should convey the name of the control for which alert message appears if user is on another control. 
- [ ] Bug9: [Accessibility][Screen Reader]: Autosuggestions are not accessible in scan mode. 
- [ ] Bug10: [Accessibility][Usable]: Narrator speaks value of the text field many times in back navigation. 
- [ ] Bug11: [Accessibility][Programmatic access]: Narrator does not convey prefix and suffix value while navigating 'Text field with a prefix'. 
- [ ] Bug12: [Accessibility][Usable]: Narrator is keep reading alert message even if user moves to the next control.  
- [ ] Bug13: [Suggestion][Accessibility][Usable]: Narrator keep saying multiple validation alerts related to the text fields when page gets loaded. 
- [ ] Bug14: [Suggestion][Accessibility][Usable]: 'Info button' should have the keyboard focus after 'Text Field with custom label render ' text field.  
  
  
Component: TextField 
@micahgodbolt

**Comment by micahgodbolt** [comment]
I think when these come in we've kept them assigned to the shield dev to work on, in hopes to spread the work out.

**Comment by TestAccessibilityGIT** [comment]
- [ ] Bug17:[Accessibility][Programmatic Access]:Screen Reader does not read the error message that is present below ‘with error message’ text field.

**Comment by Forestviolence** [comment]
@aneeshack4 @micahgodbolt On a form with multiple TextFields, validateOnBlur and tabbing to the next field then narrator will read the error message associated with the previous field after reading the label of the newly focused field. Reproes on both validateOnBlur examples here: https://developer.microsoft.com/en-us/fabric#/components/textfield

**Comment by awalia13** [comment]
@micahgodbolt - Hi Micah as 7338 has been closed, do we need to track 8226 for all the issues mentioned in 7338?
